lib/ssh_scan/database/mongo_queue.rb

```diff
@@ -0,0 +1,55 @@
+require 'mongo'
+
+Mongo::Logger.logger.level = ::Logger::FATAL
+
+module SSHScan
+  module QueueDB
+    class MongoDb
+      attr_reader :queue_collection
```

Comment:
SSHScan::JobQueue has methods that talk to SSHScan::DB

-OR-

SSHScan::DB has added methods for queuing and we drop SSHScan::JobQueue entirely